src/hooks/useProjects.ts

```diff
@@ -154,9 +33,7 @@ export function useDBProjectsQuery(
     () =>
       axios
         .get<Json<DBProjectRow>[]>(`/api/projects?${formatQueryParams(opts)}`)
-        .then(res =>
-          res.data?.map(p => parseDBProjectJson(parseDBProjectsRow(p))),
-        ),
+        .then(res => res.data?.map(parseDBProject)),
```

Comment:
just a cleanup, as previously these two functions were consistently being used in tandem